kratos/python_scripts/file_points_output_process.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+import csv, os
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# other imports
+from KratosMultiphysics.multiple_points_output_process import MultiplePointsOutputProcess
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return FilePointsOutputProcess(Model, settings["Parameters"])
+
+class FilePointsOutputProcess(KratosMultiphysics.Process):
+    """This process writes output for several points given in a file to an output file
+    Internally it holds an object of type "MultiplePointsOutputProcess"
+    Usage:
+        - file path of an .csv-file (!) for the import of sampling points need to be specified
+        - the first line of the .csv-file should state the column names: "x,y,z"
+        - the following rows should each be one point with its x,y and z value in the respective column of the .csv-file, separated by a comma 
+        - the number of sampling points needs to be specified
+    """
+    def __init__(self, model, params):
+        KratosMultiphysics.Process.__init__(self)
+
+        default_settings = KratosMultiphysics.Parameters('''{
+            "help"              : "This process writes output for several points given in a file to a file. Internally it holds an object of type MultiplePointsOutputProcess",
+            "model_part_name"   : "",
+            "entity_type"       : "element",
+            "file_path"         : "",
+            "sampling_points"   : 3,
```

Comment:
why does one need to specify this?
=> should come from the CSV file (number of rows)